### DISCUSSION TITLE
Connection Problem to database

### DISCUSSION THREAD

**Discussion by nyxongod** [supabase/supabase#2717] [Questions] [answered]
I have been trying to connect to database using your string connection of course used password for database during creation, but I can't connect --> instead this error I get --> Error: P1001: Can't reach database server at.

Command to connect --> prisma db push 

Any advice ?

Regards

**Comment by inian** [comment, marked as answer]
Hi @nyxongod, if you are still facing this issue, reach out to support@supabase.io with the api url and we will take a look. thanks!

**Comment by hp0404** [comment]
I also encountered a db connection issue when signing in from Tableau. 
In case someone else finds it, make sure to check if the host string is correct (Tableau added https:// automatically before supabase host (resulting in the error)).

**Reply by Betto325** [reply]
Hey, I'm facing a similar issue connecting supabase to tableau, could you help me?
